lib/Cake/View/Helper/FormHelper.php

```diff
@@ -1880,7 +1883,7 @@ public function postLink($title, $url = null, $options = array(), $confirmMessag
 		if (isset($options['data']) && is_array($options['data'])) {
 			foreach (Hash::flatten($options['data']) as $key => $value) {
 				$fields[$key] = $value;
-				$out .= $this->hidden($key, array('value' => $value, 'id' => false));
+				$out .= $this->hidden($key, array('value' => $value, 'id' => false, 'secure' => static::SECURE_SKIP));
```

Comment:
Are you sure these hidden fields should be skipped? Won't that make these forms fail SecurityComponent?